helm/coredns-app/values.yaml

```diff
@@ -20,10 +20,11 @@ configmap:
   log: |
     denial
     error
-  # Uncoment forward and forwadOptions to enable DNS forward https://coredns.io/plugins/forward/
+
+  # Uncoment forward and forwardOptions to enable DNS forward https://coredns.io/plugins/forward/
   # forward: . 192.168.1.1 /etc/resolv.conf
-  # forwardOptions: |
-  #   policy sequential
+  forwardOptions: |
+    policy sequential
```

Comment:
Not competent to judge, just curious do we have rationale for having forward options different from default upstream policy of `policy random`? If we do, maybe we should leave a comment here explaining that rationale, even if just for us.

Reply:
https://github.com/giantswarm/kubernetes-coredns/pull/27/files#r271379790

Reply:
It was to ensure the order but I am not sure round_robin does not make more sense

Reply:
@pipo02mix does that rationale (comment Quentin referred to) still hold? If so, Quentin please add it as comment in values.yaml that upstream default is policy random but we think policy sequential is batter default due to ...

Reply:
Still waiting for the rationale from @pipo02mix before merging :)

Reply:
Sorry for delay. The reason was the customer who requested wanted to set sequential as default. but we can make it configurable and set out own default, no blocking

Reply:
Thanks Fernando for heads up!

IIUC rather than keeping `forwardOptions` as required (when `forward` is used) and configuring our own default (even if aligned to upstream default at this moment), it might make more sense to modify the ConfigMap template to make `forwardOptions` optional so that `forwardOptions` default to upstream default automatically with every update to new upstream release; still we should keep `forwardOptions` as configurable for backward compatibility.

Reply:
@sslavic I configured it as optional now :)
I will merge after you approve